api/src/main/java/org/pmiops/workbench/config/CdrSchemaConfig.java

```diff
@@ -0,0 +1,40 @@
+package org.pmiops.workbench.config;
+
+import com.google.gson.annotations.SerializedName;
+import java.util.List;
+import java.util.Map;
+
+public class CdrSchemaConfig {
```

Comment:
nit: Would it make more sense to name this `CdrBigQuerySchemaConfig`? Both for this and the config key? More verbose, and not 100% accurate, but might help to disambiguate between this schema, and the CDR MySQL DBs.

Reply:
Done.